[2024-03-28T04:00:06.658+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_historical_congress_data.bq_create_actions_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-03-28T04:00:06.671+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_historical_congress_data.bq_create_actions_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-03-28T04:00:06.673+0000] {taskinstance.py:2193} INFO - Starting attempt 3 of 3
[2024-03-28T04:00:06.685+0000] {taskinstance.py:2217} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_actions_Congress_partitioned_table_task> on 2024-03-15 00:00:00+00:00
[2024-03-28T04:00:06.692+0000] {standard_task_runner.py:60} INFO - Started process 143 to run task
[2024-03-28T04:00:06.696+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'upload_historical_congress_data', 'bq_create_actions_Congress_partitioned_table_task', 'scheduled__2024-03-15T00:00:00+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/upload_historical_congress_data.py', '--cfg-path', '/tmp/tmp8b_753qw']
[2024-03-28T04:00:06.699+0000] {standard_task_runner.py:88} INFO - Job 184: Subtask bq_create_actions_Congress_partitioned_table_task
[2024-03-28T04:00:06.753+0000] {task_command.py:423} INFO - Running <TaskInstance: upload_historical_congress_data.bq_create_actions_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [running]> on host be5d54f7fe24
[2024-03-28T04:00:06.811+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_historical_congress_data' AIRFLOW_CTX_TASK_ID='bq_create_actions_Congress_partitioned_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T00:00:00+00:00'
[2024-03-28T04:00:06.816+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-28T04:00:06.822+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-28T04:00:06.825+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-28T04:00:06.876+0000] {bigquery.py:2811} INFO - Executing: {'query': {'query': 'CREATE OR REPLACE TABLE Congress.actions         PARTITION BY actionDate_formatted         AS         SELECT *, DATE(actionDate) actionDate_formatted         FROM Congress.actions_external_table;', 'useLegacySql': False}}'
[2024-03-28T04:00:06.879+0000] {bigquery.py:1613} INFO - Inserting job ***_upload_historical_congress_data_bq_create_actions_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_ffee059cb17db81f1e414530518f826f
[2024-03-28T04:00:08.366+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2870, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1595, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1584, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Unrecognized name: actionDate at [1:118]; reason: invalidQuery, location: query, message: Unrecognized name: actionDate at [1:118]

Location: US
Job ID: airflow_upload_historical_congress_data_bq_create_actions_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_ffee059cb17db81f1e414530518f826f

[2024-03-28T04:00:08.406+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=upload_historical_congress_data, task_id=bq_create_actions_Congress_partitioned_table_task, execution_date=20240315T000000, start_date=20240328T040006, end_date=20240328T040008
[2024-03-28T04:00:08.423+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 184 for task bq_create_actions_Congress_partitioned_table_task (400 Unrecognized name: actionDate at [1:118]; reason: invalidQuery, location: query, message: Unrecognized name: actionDate at [1:118]

Location: US
Job ID: airflow_upload_historical_congress_data_bq_create_actions_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_ffee059cb17db81f1e414530518f826f
; 143)
[2024-03-28T04:00:08.448+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-28T04:00:08.467+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
